fuel-coroutines/src/test/kotlin/com/github/kittinunf/fuel/coroutines/CoroutinesTest.kt

```diff
@@ -131,13 +192,18 @@ class CoroutinesTest {
 
     @Test
     fun testAwaitForObjectResultCatchesDeserializeError() = runBlocking {
+        mock.chain(
+            request = mock.request().withPath("/ip"),
+            response = mock.response().withBody(ByteArray(1) { 2 })
+        )
+
         try {
-            Fuel.get("/ip").awaitObjectResult(UUIDResponseDeserializer).fold({ _ ->
+            Fuel.get(mock.path("ip")).awaitObjectResult(UUIDResponseDeserializer).fold({ _ ->
                 fail("This is an error case!")
 
             }, { error ->
                 assertNotNull(error)
-                assertTrue(error.exception is JsonMappingException)
+                assertTrue(error.exception is UUIDResponseDeserializer.NoValidFormat)
```

Comment:
could we make this an assertEqual? then you get a better explanation

Reply:
Sure!